ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -128,6 +140,7 @@ public KsqlConfig(Map<?, ?> props) {
     ksqlConfigProps.put(KSQL_PERSISTENT_QUERY_NAME_PREFIX_CONFIG, KSQL_PERSISTENT_QUERY_NAME_PREFIX_DEFAULT);
     ksqlConfigProps.put(KSQL_TRANSIENT_QUERY_NAME_PREFIX_CONFIG, KSQL_TRANSIENT_QUERY_NAME_PREFIX_DEFAULT);
     ksqlConfigProps.put(KSQL_TABLE_STATESTORE_NAME_SUFFIX_CONFIG, KSQL_TABLE_STATESTORE_NAME_SUFFIX_DEFAULT);
+    ksqlConfigProps.put(KSQL_QUERY_CLOSE_WAIT_TIME_CONFIG, KSQL_QUERY_CLOSE_WAIT_TIME_DEFAULT);
```

Comment:
I may be missing something, but I don't see a `get` for this property. Is it being used? If so, how?

Reply:
@apurvam This is not used anymore, I went back to use close() with no args so there is no need for the wait and therefore no need for this config variable.

Reply:
Then we should have removed the configuration before merging. Can you submit a follow up PR? 

Also, did you see my comments about commenting out the assertions which were failing in Jenkins? we should fix understand the cause and fix it, or at least understand the cause and document it in a JIRA .